<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordGenerator</class>
 <widget class="QMainWindow" name="PasswordGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PasswordGenerator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>480</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="mainGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="checkBoxLayouts">
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Majuscule</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="Lowercase">
             <property name="text">
              <string>Minuscule</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>Chiffres</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>Méta-Caractères</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Taille du mot de passe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="TailleMDP">
           <property name="font">
            <font>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background:none;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>32</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ValeurTailleMdp">
           <property name="font">
            <font>
             <family>Bitstream Vera Sans Mono</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Générer le mot de passe</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QPushButton" name="CopierMdp">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Copier</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="Resultats">
           <property name="font">
            <font>
             <family>Bitstream Vera Sans Mono</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>TailleMDP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ValeurTailleMdp</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
